.teamcity/_self/projects/WebApp.kt

```diff
@@ -89,6 +102,7 @@ object BuildDockerImage : BuildType({
 					registry.a8c.com/calypso/app:build-%build.number%
 					registry.a8c.com/calypso/app:commit-${Settings.WpCalypso.paramRefs.buildVcsNumber}
 					registry.a8c.com/calypso/app:latest
+					registry.a8c.com/calypso:${Settings.WpCalypso.paramRefs.buildVcsNumber}-%teamcity.build.branch%
```

Comment:
Needed so we can conditionally push trunk (matches current image name).

Reply:
If possible, I'd like to keep the name pattern `registry.a8c.com/calypso/app:<label>`. What about `registry.a8c.com/calypso/app:latest-%teamcity.build.branch%`? That way we can always have `registry.a8c.com/calypso/app:latest-trunk`, but we'll also produce `registry.a8c.com/calypso/app:latest-<branch-name>` in case someone wants to download the latest available image for their branch.

Reply:
I'm matching the tag names with the current process, so we can migrate it easily for now. Can change it in a subsequent release.